draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -248,23 +248,26 @@ check-list.
 explanation of the deviations from {{!RFC2914}}.
 
 (1)
-: Impact on Standard TCP, SCTP {{!RFC2960}}, and DCCP {{!RFC4340}}.
+: Impact on existing deployments of TCP {{!RFC9293}}, SCTP {{!RFC9260}},
+DCCP {{!RFC4340}}, and QUIC {{!RFC9000}}.
 
 : Proposed congestion control mechanisms should be evaluated when
-competing with standard IETF congestion control {{!RFC2581}},
-{{!RFC2960}}, {{!RFC4340}}.  Alternate congestion controllers that have a
-significantly negative impact on traffic using standard
-congestion control may be suspect and this aspect should be part
-of the community's decision making with regards to the
-suitability of the alternate congestion control mechanism.
-
-: We note that this bullet is not a requirement for strict TCP-
+competing with standard IETF congestion control {{!RFC5681}},
+{{!RFC9260}}, {{!RFC4340}}, {{!RFC9002}}, {{!RFC9438}}.  Alternate
+congestion controllers that have a significantly negative impact on
+traffic using standard congestion control may be suspect and this aspect should
+be part of the community's decision making with regards to the suitability of
+the alternate congestion control mechanism. The community should also consider
+any other non-standard congestion controls shown to be widely deployed,
+
```

Comment:
I sympathise with the non-standard statement, but I would not like a pushback that I have to evaluate against something for which there is no current open specification - after trying to evaluate against some of these, I have learned this is a fool's game. Do we need to say it as a should?

Reply:
Well, "non-standard" != "not specified". Secondly, it already says "should" so I'm not sure what your objection is?

Reply:
That last sentence is so awkward: I'm just unsure how someone would evaluate against something that's widely deployed, but not fully specified - you can't simulate it, or do any mathematical analysis only observe a specific version of the traffic, and if that changes, you might not know if the analysis is still correct?

Reply:
A minimal fix could be: /any other/other/ and /shown to be/known to be/ ... that helps because "consider" is weak anyway;-).

Reply:
OK, I'm going to make this "minimal fix" and then argue about non-standard algos in another issue.